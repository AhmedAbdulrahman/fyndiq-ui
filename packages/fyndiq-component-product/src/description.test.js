import React from 'react'
import { shallow } from 'enzyme'

import Description from './description'

describe('Product Description', () => {
  it('should have a default structure', () => {
    expect(
      shallow(<Description>description content</Description>),
    ).toMatchSnapshot()
  })

  it('should accept nodes', () => {
    expect(
      shallow(
        <Description>
          <strong>description</strong>
          <em>test</em>
        </Description>,
      ),
    ).toMatchSnapshot()
  })
})
