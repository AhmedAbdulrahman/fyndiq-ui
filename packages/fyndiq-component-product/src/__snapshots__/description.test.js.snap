// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product Description should accept nodes 1`] = `
<div
  className="description"
>
  <strong>
    description
  </strong>
  <em>
    test
  </em>
</div>
`;

exports[`Product Description should have a default structure 1`] = `
<div
  className="description"
>
  description content
</div>
`;
